aiida_quantumespresso/calculations/namelists.py

```diff
@@ -5,76 +5,67 @@
 afterwards).
 """
 from __future__ import absolute_import
+
 import os
-from aiida.common import InputValidationError
-from aiida.common import CalcInfo, CodeInfo
+import six
+
+from aiida.common import datastructures, exceptions
+from aiida.engine import CalcJob
 from aiida.orm import Dict, Code
 from aiida.orm import RemoteData, FolderData, SinglefileData
-from aiida.engine import CalcJob
-from aiida.common import CodeInfo
 from aiida.plugins import ParserFactory
+
 from aiida_quantumespresso.calculations import _lowercase_dict, _uppercase_dict
 from aiida_quantumespresso.utils.convert import convert_input_to_namelist_entry
-import six
 
-# TODO: review usage of class variables and input names
-# TODO: add exit codes
```

Comment:
My idea here was to move some exit codes in the base class, to enforce max consistency of exit codes across subclasses. Bad idea?

Reply:
No good, idea! But we should not do this *just* for the `NamelistsCalculation` but we should homogenize as much as possible for all calculations. So let's open an issue and tackle this once all classes are migrated